internal/oci/util.go

```diff
@@ -16,3 +16,9 @@ func IsWCOW(s *specs.Spec) bool {
 func IsIsolated(s *specs.Spec) bool {
 	return IsLCOW(s) || (s.Windows != nil && s.Windows.HyperV != nil)
 }
+
+// IsJobContainer checks if `s` is asking for a Windows job container.
+func IsJobContainer(s *specs.Spec) bool {
+	val, ok := s.Annotations[AnnotationHostProcessContainer]
+	return val == "true" && ok
```

Comment:
shouldn't you check if `ok` is true first?

Reply:
I don't even think we need to check ok at all really, if the values not in the map we'd get an empty string for val so just checking if val == "true" should suffice